modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,20 +14,41 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-The following issue has been resolved in the current release:
+The following issues have been resolved in the current release:
+//current release is 2.5.1/2.4.7/2.3.11 scheduled for April 18, 2024
 
-* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
+* https://issues.redhat.com/browse/OSSM-6177[OSSM-6177] Previously, when validation messages were enabled in the `ServiceMeshControlPlane` (SMCP), the `istiod` crashed continuously unless `GatewayAPI` support was enabled. Now, when validation messages are enabled but `GatewayAPI` support is not, the `istiod` does not continuously crash.
+
+* https://issues.redhat.com/browse/OSSM-6163[OSSM-6163] Resolves the following:
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-6163[OSSM-6163] Resolves the following issues:
```
Per ISG, "To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun."